src/index.js

```diff
@@ -47,10 +47,14 @@ const toJSON = (obj) => JSON.parse(JSON.stringify(obj))
 export default function (schema, opts) {
   const options = merge({}, defaultOptions, opts)
 
+  // get _id type from schema
+  options._idType = schema.tree._id.type
```

Comment:
Yes, unless someone ruins the `_id` property..

A simple example:
```js
const mySchema = new mongoose.Schema({
  _id: {},
  ...
})
```

That's quite out of this plugins scope though, the assertion could be removed quite happily.